src/main.py

```diff
@@ -132,6 +140,8 @@ def main():
         spawn_water_lanes(frame_count, water_lane1, water_lane2, water_lane3, water_lane4, water_lane5,
                           render_group, asset_dict, WIN)
         draw_sprites(render_group, WIN, background)
+        # draw_sprites(win_group, WIN, background)
+        # draw_sprites(kill_group, WIN, background)
```

Comment:
I'd prefer if you got rid of these. I'm sure you used them for figuring out the placements but I'd rather not have them there to accidentally get uncommented.

Reply:
Oops, yeah meant to remove those, mb.